### PR TITLE
Re-enable w7900 jobs.

### PR DESCRIPTION
**Pull request by saienduri**
Enabling w7900 jobs as runner should work as intended now. Progress on https://github.com/iree-org/iree/issues/17370

ci-extra: test_amd_w7900

### CODE REVIEW

**Comment by ScottTodd** [comment]
For the (new) failing DCO check, see https://iree.dev/developers/general/contributing/#developer-certificate-of-origin

File: build_tools/pkgci/external_test_suite/pytorch_models_cpu_llvm_task.json [hunk 1]
```diff
@@ -14,7 +14,9 @@
   "expected_compile_failures": [
     // TODO(#17344): need to regenerate .mlirbc
     "opt-125M",
-    "resnet50",
+    "resnet50"
   ],
-  "expected_run_failures": []
+  "expected_run_failures": [
+    "sdxl-vae-decode-tank"
+  ]
```
Comment (ScottTodd) [inline build_tools/pkgci/external_test_suite/pytorch_models_cpu_llvm_task.json:21]: Aw, not great that this regressed on CPU

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -429,45 +429,44 @@ jobs:
...
+      BUILD_DIR: build-tests
+      INSTALL_DIR: ${{ needs.build_all.outputs.install-dir }}
+      INSTALL_DIR_ARCHIVE: ${{ needs.build_all.outputs.install-dir-archive }}
+      INSTALL_DIR_GCS_URL: ${{ needs.build_all.outputs.install-dir-gcs-url }}
+      IREE_CPU_DISABLE: 1
+      IREE_VULKAN_DISABLE: 0
+      IREE_CUDA_DISABLE: 1
+      IREE_HIP_DISABLE: 0
+      IREE_HIP_TEST_TARGET_CHIP: "gfx1100"
+    runs-on: nodai-amdgpu-w7900-x86-64
```
Comment (ScottTodd) [inline .github/workflows/ci.yml]: When will we have more than one runner? Even when the runner was relatively stable, we still saw long queue times with how many jobs there are now.

File: build_tools/pkgci/external_test_suite/pytorch_models_cpu_llvm_task.json (hunk 1 above)
Comment (saienduri) [reply build_tools/pkgci/external_test_suite/pytorch_models_cpu_llvm_task.json:21]: yeah vae-decode and scheduled-unet on cpu regressed :(

File: .github/workflows/ci.yml (hunk 2 above)
Comment (saienduri) [reply .github/workflows/ci.yml]: In the next couple of days (fingers crossed). pushing on getting that done

**Comment by ScottTodd** [comment]
I added `ci-extra: test_amd_w7900` to the PR description since that is postsubmit only. May want to retrigger CI to ensure that didn't regress after being disabled

**Comment by saienduri** [comment]
Looks like the following tests failed:
	 54 - iree/hal/drivers/hip/dynamic_symbols_test (Failed)
Should we add an XFail for this test somewhere?

**Comment by ScottTodd** [comment]
> Looks like the following tests failed: 54 - iree/hal/drivers/hip/dynamic_symbols_test (Failed) Should we add an XFail for this test somewhere?

No. That's a baseline test for the HIP driver. If that fails all bets for the rest of the code are off. Need to fix the test or code.

**Comment by saienduri** [comment]
> > Looks like the following tests failed: 54 - iree/hal/drivers/hip/dynamic_symbols_test (Failed) Should we add an XFail for this test somewhere?
> 
> No. That's a baseline test for the HIP driver. If that fails all bets for the rest of the code are off. Need to fix the test or code.

Ok makes sense. I will let Erman know that test is failing and disable that for now then

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -428,8 +428,8 @@ jobs:
           IREE_HIP_DISABLE: 0
         run: |
           ./build_tools/cmake/ctest_all.sh ${BUILD_DIR}
-
-  # TODO(#17370): re-enable when nodai-amdgpu-w7900-x86-64 runner is stable
+  
+  # tests failed: 54 - iree/hal/drivers/hip/dynamic_symbols_test (Failed)
```
Comment (ScottTodd) [inline .github/workflows/ci.yml]: Trailing spaces. Could also keep the issue TODO but change the context

**Review by ScottTodd — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (ScottTodd) [reply .github/workflows/ci.yml]: :/ yeah, already seeing some 45+ minute queues.

**Pull request merged**